rally_ovs/plugins/ovs/ovsclients.py

```diff
@@ -172,6 +172,29 @@ def parse_lswitch_list(lswitch_data):
         lswitches.append({"name": line.split(" ")[1].strip('()')})
     return lswitches
 
+def get_lb_info(info):
+    lbs = {}
+
+    lb_name = None
+    for line in info.splitlines():
+        line = line.strip()
+        if len(line) == 0:
+            continue
+        
+        if not lb_name:
+            lb_name = line
+        else:
+            tokens = line.split('=')
+            if len(tokens) == 0:
+                lb_name = None
+                continue
+            lb_vip = tokens[0]
+            lb_backends = tokens[1] if len(tokens) == 2 else ''
+            lbs[lb_name] = (lb_vip, lb_backends)
+            lb_name = None
+
```

Comment:
If the `=` token is not found, then `split()` will return a list containing the whole string as the first item. Therefore, `len(tokens)` wont' ever be 0.

You can just replace this whole block with
```
lbs[lb_name] = tuple(line.split('=', 2))
lb_name = None
```
This will ensure that `lbs[lb_name]` is set to a tuple of 2 items. If the `=` is not present, then the second item of the tuple will be an empty string.

Reply:
Way better, will do, thanks!